spec/registry.yaml

```
@@ -86,3 +86,8 @@ groups:
         note: >
           The coordinate follows the format "{ParentType}.{fieldName}" and uniquely
           identifies the field within the GraphQL schema.
+      - id: graphql.operation.step.id
+        brieg: "The id of the step in the execution plan."
+        type: int
```

Must the id be an integer? Grafast currently uses numeric step IDs but it reserves space for IDs to be strings in future. Strings allow for more stable plans, since a small change won't need to re-number every step - and the id can include details of what it's doing like `NullCheck-1` or whatever. Also, if your ids are numbers then parsing them from string should be straightforward, but the other way around is more complex.

```suggestion
        type: string
```